2018-08-14-scrum_erklaersession_cgerigk.md

```diff
@@ -0,0 +1,27 @@
+---
+title:     Scrum in einfachen Worten und Bildern erklären
+author:    Christiane Gerigk
+published: 14.08.2018
+image:     Titel-Scrum-in-einfachen-Worten-320x202.jpg, Christiane-Gerigk-320x202.png
```

Comment:
The preview picture for the blog article list can currently only be one in the new blog. I would remove Christiane's photo then.